dblookupext/miniblocksHandler.go

```diff
@@ -0,0 +1,198 @@
+package dblookupext
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+	"github.com/multiversx/mx-chain-core-go/hashing"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	"github.com/multiversx/mx-chain-go/storage"
+)
+
+type miniblocksHandler struct {
+	marshaller                       marshal.Marshalizer
+	hasher                           hashing.Hasher
+	epochIndex                       *epochByHashIndex
+	miniblockHashByTxHashIndexStorer storage.Storer
+	miniblocksMetadataStorer         storage.Storer
+}
+
+// blockCommitted will record the miniblocks information
+func (mh *miniblocksHandler) blockCommitted(header data.HeaderHandler, blockBody *block.Body) error {
+	headerHash, errCalculate := core.CalculateHash(mh.marshaller, mh.hasher, header)
+	if errCalculate != nil {
+		return fmt.Errorf("%w for header, header round %d", errCalculate, header.GetRound())
+	}
+
+	for index, mb := range blockBody.MiniBlocks {
+		miniblockHash, err := core.CalculateHash(mh.marshaller, mh.hasher, mb)
```

Comment:
L31-34 can be moved inside `handleMiniblock` func

Reply:
this will overcomplicate the returning errors on handleMiniblock as I want to know the miniblock hash when wrapping the error on L38